src/libslic3r/PrintConfig.cpp

```diff
@@ -3676,30 +3691,23 @@ void PrintConfigDef::init_fff_params()
     def->mode = comAdvanced;
     def->set_default_value(new ConfigOptionFloats{ 0., 0. });
 
-    // resonance avoidance ported over from qidi slicer
+
+
+   // resonance avoidance ported over from qidi slicer
     def          = this->add("resonance_avoidance", coBool);
     def->label   = L("Resonance avoidance");
     def->tooltip = L("By reducing the speed of the outer wall to avoid the resonance zone of the printer, ringing on the surface of the "
-                     "model are avoided.\n"
+                     "model is avoided.\n"
                      "Please turn this option off when testing ringing.");
     def->mode    = comAdvanced;
     def->set_default_value(new ConfigOptionBool(false));
 
-    def           = this->add("min_resonance_avoidance_speed", coFloat);
-    def->label    = L("Min");
-    def->tooltip  = L("Minimum speed of resonance avoidance.");
-    def->sidetext = "mm/s";	// milimeters per second, don't need translation
-    def->min      = 0;
-    def->mode     = comAdvanced;
-    def->set_default_value(new ConfigOptionFloat(70));
+    def          = this->add("resonance_avoidance_speed_ranges", coString);
```

Comment:
As a workaround, you can use ConfigOptionPoints to avoid parsing strings. One 2d point is basically a range